crates/bitwarden-api-key-connector/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "bitwarden-api-key-connector"
+description = "Api bindings for the Bitwarden Key Connector API."
+categories = ["api-bindings"]
+
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+homepage.workspace = true
+repository.workspace = true
+license-file.workspace = true
+keywords.workspace = true
+
+[dependencies]
+async-trait = { workspace = true }
+mockall = { version = ">=0.13.1, <0.15" }
```

Comment:
🎨  mockall (0.14.0) is also present in the autotype-crate. Make sense to move this up to the root level and use the same version across both crates